source/slang/lexer.h

```diff
@@ -100,13 +92,16 @@ namespace Slang
         char const*     begin;
         char const*     end;
 
+        /// The starting sourceLoc (same as first location of SourceUnit)
+        SourceLoc       startLoc;           
+
         // The starting source location for the code as written,
         // which cannot be overridden.
-        SourceLoc       spellingStartLoc;
+        //SourceLoc       spellingStartLoc;
```

Comment:
Don't forget to remove the commented-out declarations (and their descriptions) before checkin!